tests/database.yml

```diff
@@ -0,0 +1,38 @@
+Databases:
+   VoxCeleb:
+      - /path/to/voxceleb/voxceleb1/dev/wav/{uri}.wav
+      - /path/to/voxceleb/voxceleb1/test/wav/{uri}.wav
+      - /path/to/voxceleb/voxceleb2/dev/aac/{uri}.wav
+      - /path/to/voxceleb/voxceleb2/test/aac/{uri}.wav
+   MUSAN: /path/to/musan/{uri}.wav
+   AMI: /path/to/amicorpus/*/audio/{uri}.wav
+   MyVoxCeleb:
+      - /path/to/voxceleb/voxceleb1/dev/wav/{uri}.wav
+      - /path/to/voxceleb/voxceleb1/test/wav/{uri}.wav
+      - /path/to/voxceleb/voxceleb2/dev/aac/{uri}.wav
+      - /path/to/voxceleb/voxceleb2/test/aac/{uri}.wav
+
+Protocols:
+   AMI:
+      SpeakerDiarization:
+         MixHeadset:
+           train:
+              annotation: /path/to/AMI/MixHeadset.train.rttm
+              annotated: /path/to/AMI/MixHeadset.train.uem
+           development:
+              annotation: /path/to/AMI/MixHeadset.development.rttm
+              annotated: /path/to/AMI/MixHeadset.development.uem
+           test:
+              annotation: /path/to/AMI/MixHeadset.test.rttm
+              annotated: /path/to/AMI/MixHeadset.test.uem
+   MyVoxCeleb:
+      SpeakerVerification:
+         VoxCeleb2:
+           train:
+              annotation: /path/to/voxceleb/myvoxceleb/VoxCeleb2.train.rttm
+            #   annotated: /path/to/voxceleb/myvoxceleb/VoxCeleb2.train.uem
+           test_trial:
+              # annotation: /path/to/voxceleb/myvoxceleb/VoxCeleb2.test.rttm
+              # annotated: /path/to/voxceleb/myvoxceleb/VoxCeleb2.test.uem
+              duration: /path/to/voxceleb/myvoxceleb/vox1_all_duration.txt.gz # should contain all the file present in the trial
+              trial: /path/to/voxceleb/myvoxceleb/VoxCeleb2.test.trial 
```

Comment:
Would you mind pushing a sample `.trial` file for me to get a better idea of what the code does?

Reply:
Here is the first lines of my ```Voxceleb2.test.trial``` :
```
1 id10270/x6uYqmx31kE/00001.wav id10270/8jEAjG6SegY/00008.wav
0 id10270/x6uYqmx31kE/00001.wav id10300/ize_eiCFEg0/00003.wav
1 id10270/x6uYqmx31kE/00001.wav id10270/GWXujl-xAVM/00017.wav
0 id10270/x6uYqmx31kE/00001.wav id10273/0OCW1HUxZyg/00001.wav
1 id10270/x6uYqmx31kE/00001.wav id10270/8jEAjG6SegY/00022.wav
0 id10270/x6uYqmx31kE/00001.wav id10284/Uzxv7Axh3Z8/00001.wav
1 id10270/x6uYqmx31kE/00001.wav id10270/GWXujl-xAVM/00033.wav
0 id10270/x6uYqmx31kE/00001.wav id10284/7yx9A0yzLYk/00029.wav
1 id10270/x6uYqmx31kE/00002.wav id10270/5r0dWxy17C8/00026.wav
0 id10270/x6uYqmx31kE/00002.wav id10285/m-uILToQ9ss/00009.wav
```

It is simply a rename of the file ```verif_original.txt.gz``` (that I uncompressed) present in ```pyannote.db.voxceleb```.